shared/constant/constant.go

```diff
@@ -14,12 +14,13 @@
 
 package constant
 
+// PrivilegedPlugins can be changed by 'WOODPECKER_ESCALATE' at runntime
 var PrivilegedPlugins = []string{
-	"plugins/docker",
-	"plugins/gcr",
-	"plugins/ecr",
-	"woodpeckerci/plugin-docker",
-	"woodpeckerci/plugin-docker-buildx",
+	"docker.io/plugins/docker:20.14.0",
+	"docker.io/plugins/gcr:20.14.0",
+	"docker.io/plugins/ecr:20.14.0",
+	"docker.io/woodpeckerci/plugin-docker-buildx:2.1.0",
+	// "docker.io/woodpeckerci/plugin-docker",
```

Comment:
I am not sure if this list support tags and if we should pin those images as they are used for checking the permissions and not for fetching images.

Reply:
This will be hard to manage to have to count all allowed tags for each image

Reply:
Also I don't see how that improves security as if you can push new tag you can also push malicious tag over existing version tag. For security reasons we would have to pin to image tag sha256 than